src/systems/cursor-targetting-system.js

```diff
@@ -25,6 +27,21 @@ export class CursorTargettingSystem {
     this.rightRemote.components["cursor-controller"].tick2(t);
   }
 
+  onObject3DSet() {
+    this.dirty = true;
+  }
+
+  onObject3DRemove() {
+    this.dirty = true;
+  }
+
+  onMutation(records) {
+    // let's try to avoid re-querying the targets on attribute changes we know we can ignore
+    if (records.some(r => r.type === "childList" || (r.type === "attributes" && r.attributeName === "class"))) {
```

Comment:
MDN does not know whether desktop chrome, firefox, or safari support `MutationRecord.type`, so I suggest putting a breakpoint here on each of those browsers just to make sure `r.type` works as expected.
https://developer.mozilla.org/en-US/docs/Web/API/MutationRecord

Reply:
^ This comment is out of date. This looks to be supported across all browsers.